pkger/models.go

```diff
@@ -887,10 +906,12 @@ func (c chart) validProperties() []ValidationErr {
 	switch c.Kind {
 	case chartKindGauge:
 		fails = append(fails, c.Colors.hasTypes(colorTypeMin, colorTypeThreshold, colorTypeMax)...)
-	case chartKindScatter:
-		fails = append(fails, c.Axes.hasAxes("x", "y")...)
 	case chartKindHeatMap:
 		fails = append(fails, c.Axes.hasAxes("x", "y")...)
+	case chartKindHistogram:
+		fails = append(fails, c.Axes.hasAxes("x")...)
```

Comment:
acc'd the UI and the histogram datamodel, histogram charts only requires an x-axis. I could be mistaken